CHANGELOG.md

```diff
@@ -1,8 +1,18 @@
 # PD Change Log
+
+## v2.0.0-GA
+### New Features
+* Support using pd-ctl to scatter spcified Regions for manually adjusting hotspot Regions in some cases
+### Improvements
+* More configuration check rules to prevent unreasonable configuration
```

Comment:
Improve the configuration check rules to prevent unreasonable scheduling configuration